edge_computing/image_base_install/ibi-understanding-image-based-install.adoc

```diff
@@ -0,0 +1,127 @@
+
+* The `install-config.yaml` installation manifest
+
+* The `image-based-config.yaml` manifest
+
+The `openshift-install` program uses these resources to generate a configuration ISO that you attach to the preinstalled target host to complete the deployment.
+
+Managed {sno} cluster deployment::
+
+{rh-rhacm-first} and the multicluster engine for Kubernetes operator (MCE) use a hub-and-spoke architecture to manage and deploy {sno} clusters across multiple sites. Using this approach, the hub cluster serves as a central control plane that manages the spoke clusters, which are often remote {sno} clusters deployed at the far edge of the network.
++
+You can define the site-specific configuration resources for an image-based deployment in the hub cluster. The IBI Operator uses these configuration resources to reconfigure the preinstalled host at the remote site and deploy the host as a managed {sno} cluster. This approach is especially beneficial for telecommunications providers and other service providers with extensive, distributed infrastructures, where an end-to-end installation at the remote site would be time-consuming and costly.
++
+The following is a high-level overview of the image-based deployment process for hosts preinstalled with an imaged-based installation:
++
+* Define the site-specific configuration resources for the preinstalled host in the hub cluster.
+* Apply these resources in the hub cluster. This initiates the deployment process.
+* The IBI Operator creates a configuration ISO.
+* The IBI Operator boots the target preinstalled host with the configuration ISO attached.
+* The host mounts the configuration ISO and begins the reconfiguration process.
+* When the reconfiguration completes, the {sno} cluster is ready.
++
+As the host is already preinstalled using an image-based installation, a technician can reconfigure and deploy the host in a matter of minutes.
+
+[id="ibi-installation-deployment-components"]
+== Image-based installation and deployment components
+
+The following content describes the components in an image-based installation and deployment.
+
+Seed image:: OCI container image generated from a dedicated cluster with the target {product-title} version. 
+
+Seed cluster:: Dedicated {sno} cluster that you use to create a seed image and is deployed with the target {product-title} version.
+
+{lcao}:: Generates the seed image.
+
+Image Based Install (IBI) Operator:: When you deploy managed clusters, the IBI Operator creates a configuration ISO from the site-specific resources you define in the hub cluster, and attaches the configuration ISO to the preinstalled host by using a bare-metal provisioning service.
+
+`openshift-install` program:: Creates the installation and configuration ISO, and embeds the seed image URL in the live installation ISO. Attach the configuration ISO to a preinstalled host to complete the deployment.
```

Comment:
openshift-install doesn't attach the configuration ISO. IMHO we should adjust the sentence so it's clear.

```suggestion
`openshift-install` program:: Creates the installation and configuration ISO, and embeds the seed image URL in the live installation ISO. 
If IBIO is not used, manually attach the configuration ISO to a preinstalled host to complete the deployment.
```